TX/Radiomaster T8L.json

```diff
@@ -0,0 +1,21 @@
+{
+    "serial_rx":3,
+    "serial_tx":1,
+    "radio_busy":5,
+    "radio_dio1":4,
+    "power_rxen":9,
+    "power_txen":10,
+    "radio_miso":12,
+    "radio_mosi":13,
+    "radio_nss":15,
+    "radio_rst":2,
+    "radio_sck":14,
+    "power_min":0,
+    "power_high":3,
+    "power_max":3,
+    "power_default":3,
+    "power_control":0,
+    "power_values":[-10,-5,-2,0],
+    "power_lna_gain": 0,
```

Comment:
I'm pretty sure the LNA gain should be 12. It using the RFX2401C/AT2401C

Reply:
Thanks, PK! We had asked RM and were waiting to hear back